ecosystem/typescript/sdk_v2/src/transactions/authenticator/account.ts

```diff
@@ -82,3 +83,34 @@ export class AccountAuthenticatorMultiEd25519 extends AccountAuthenticator {
     return new AccountAuthenticatorMultiEd25519(public_key, signature);
   }
 }
+
+export class AccountAuthenticatorSecp256k1 extends AccountAuthenticator {
+  public readonly public_key: Secp256k1PublicKey;
+
+  public readonly signature: Secp256k1Signature;
+
+  constructor(public_key: Secp256k1PublicKey, signature: Secp256k1Signature) {
+    super();
+    this.public_key = public_key;
+    this.signature = signature;
+  }
+
+  /**
+   * Transaction authenticator Secp256k1 for a single signer transaction
```

Comment:
```suggestion
   * A Secp256k1 AccountAuthenticator for a single signer
```

or something like this- technically this isn't the `TransactionAuthenticator`, it's the `AccountAuthenticator`